### PR TITLE
dead maps

### PR DESCRIPTION
**Pull request by Unnunu**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀 This PR matches 6 functions (+0.07%) on `us`.

✅ This PR fixes 12 warnings!

File: include/script_api/macros.h [hunk 1]
```diff
@@ -568,6 +568,11 @@
 /// The given arguments can be accessed from the API function using `thread->ptrReadPos`.
 #define EVT_CALL(FUNC, ARGS...)                     EVT_CMD(EVT_OP_CALL, (Bytecode) FUNC, ##ARGS),
 
+/// Unknown cmd 90
+#define EVT_90(UNK)                             EVT_CMD(EVT_OP_90, UNK),
+
+/// Prints debug message
+#define EVT_DEBUG_PRINT(UNK)                    EVT_CMD(EVT_OP_DEBUG_PRINT, UNK),
```
Comment (HailSanta) [inline include/script_api/macros.h:575]: We can probably call these:
EVT_90 --> EVT_DEBUG_PRINT_STR or EVT_DEBUG_LOG, bases on usage, obviously its not implemented in the release build (see func_802C6E14).
EVT_DEBUG_PRINT --> EVT_DEBUG_PRINT_VAR

File: src/world/dead/area_flo/flo.h [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    AF_FLO_RaisedRedPillar              = AreaFlag(41),
+    AF_FLO_RaisedGreenPillar            = AreaFlag(42),
+    AF_FLO_LoweredPurplePillar          = AreaFlag(43),
+    AF_FLO_RidingBeanstalk              = AreaFlag(44),
+    AF_FLO_BeanstalkFadedOut            = AreaFlag(45),
+};
+
+#endif
+
+/*
```
Comment (ethteck) [inline src/world/dead/area_flo/flo.h:64]: can/should this be removed? or is it here for the future

**Review by ethteck — COMMENTED** [review]
Great work :) nice to see this stuff all cleaned up

File: src/world/dead/area_flo/flo.h (hunk 2 above)
Comment (Unnunu) [reply src/world/dead/area_flo/flo.h]: Now it's removed

**Pull request merged**